### PR TITLE
Fixes #29195 - puppet-certs should run on el8

### PR DESCRIPTION
**Pull request by wbclark**

### CODE REVIEW

**Comment by ehelms** [comment]
I think this will need katello-certs-tools package for EL8 to pass. Will work on that.

File: spec/spec_helper_acceptance.rb [hunk 1]
```diff
@@ -22,14 +22,20 @@
...
         on host, 'yum clean expire-cache'
+
+        major = fact_on(host, 'os.release.major')
+
+        if major == '8'
+          on host, puppet('resource', 'package', 'glibc-langpack-en', 'ensure=installed')
+        end
         # foreman nightly provides katello-cert-tools
-        host.install_package('https://yum.theforeman.org/releases/nightly/el7/x86_64/foreman-release.rpm')
+        host.install_package('https://yum.theforeman.org/releases/nightly/el#{major}/x86_64/foreman-release.rpm')
```
Comment (ekohl) [inline spec/spec_helper_acceptance.rb:38]: Single quotes don't allow variable interpolation
```suggestion
        host.install_package("https://yum.theforeman.org/releases/nightly/el#{major}/x86_64/foreman-release.rpm")
```

**Comment by ekohl** [comment]
This misses metadata.json

**Comment by wbclark** [comment]
> This misses metadata.json

Thanks, I've opened https://github.com/theforeman/puppet-certs/pull/281 to address it

**Pull request merged**